src/test/java/com/adobe/epubcheck/test/command_line_Test.java

```diff
@@ -233,7 +233,7 @@ public void xmpfile_Test()
   public void failonwarnings_Test()
   {
     String[] extraArgs = {"--failonwarnings"};
-    runExtraCommandLineArgTest("failonwarnings", 1, extraArgs);
+    runExtraCommandLineArgTest("failonwarnings", 0, extraArgs);
```

Comment:
Doesn't this change remove the usefulness of this test, which if I understand correctly checks that EPUBCheck fails when a `WARNING` is reported?

Should we use the "remove the title" trick here too?

Reply:
Oops, ya, I missed that it was also testing warning. Will fix.